sktime/forecasting/timesfm_forecaster.py

```diff
@@ -20,6 +20,45 @@ class TimesFMForecaster(_BaseGlobalForecaster):
     developed by Google Research for time-series forecasting. This method has been
     proposed in [2]_ and official code is given at [1]_.
 
+    TimesFM can be used either as a locally maintained package in sktime or directly
+    from the source package, allowing users to leverage either their own
+    environment or the latest updates from the source package `timesfm`.
+
+    The class offers two key flag for handling dependencies and source package behavior:
+    - `use_source_package`: Determines whether to use the source package `timesfm`.
+    - `ignore_deps`: Controls whether dependencies are enforced or ignored,
+    enabling users to bypass dependency checks and manage their environment manually.
+
+    Changes in Dependency Management:
```

Comment:
no need to add a changelog in the docstring.

Since no current behaviour changes afaik, you can just leave the description of the new behaviour, no?